frontend/src/components/featured-post-card.tsx

```diff
@@ -5,10 +5,14 @@ import CategoryPill from '@/components/category-pill';
 
 export default function FeaturedPostCard({ post }: { post: Post }) {
   const navigate = useNavigate();
+  function createSlug(title:string): string {
+    return title.toLowerCase().replace(/\s+/g, '-').replace(/[^\w-]+/g, '');
+  }
+  const slug = createSlug(post.title);
   return (
```

Comment:
Can we create a util for this, as i see it is getting used in every component, it would be better if we could reuse the function.